sormas-app/gradle.properties

```diff
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
Why did this have to be removed? Doesn't it mean that the default value of 256m is used?

Reply:
Java 8 VM no longer supports `MaxPermSize`. It doesn't work locally for me and I really don't get how this gets through the gradle CI as it uses openjdk 11...

Reply:
done